[CHANGE] ui option name for feature file boundaries

singjc opened the pull request:
Change peak picking option OSW-PyProphet to Feature File Boundaries
<!--- START AUTOGENERATED NOTES --->
# Contents ([#99](https://github.com/Roestlab/massdash/pull/99))

### Other
- Feature File Peak Boundaries for raw GUI workflow
- Roestlab/massseer into patch/ui_rename_results_peak_boundaries

### Uncategorised!
- [CHANGE] ui option name for feature file boundaries
- [RENAME] feature file peak pickign method

<!--- END AUTOGENERATED NOTES --->

singjc committed:
[CHANGE] ui option name for feature file boundaries

jcharkow commented on massdash/server/PeakPickingServer.py:
```diff
@@ -35,7 +35,7 @@ def __init__(self, peak_picking_settings: PeakPickingUISettings, chrom_plot_sett
 
     def perform_osw_pyprophet_peak_picking(self, xic_data: SqMassLoader, transition_list_ui: Literal['ExtractedIonChromatogramAnalysisUI', 'RawTargetedExtractionAnalysisUI']):
```
change method name as well?

jcharkow commented on massdash/server/PeakPickingServer.py:
```diff
@@ -35,7 +35,7 @@ def __init__(self, peak_picking_settings: PeakPickingUISettings, chrom_plot_sett
 
     def perform_osw_pyprophet_peak_picking(self, xic_data: SqMassLoader, transition_list_ui: Literal['ExtractedIonChromatogramAnalysisUI', 'RawTargetedExtractionAnalysisUI']):
```
I am having difficulties with this method in the GUI could you please look into this? I think that the loader object for the raw data is not currently being passed which causes loadTransitionGroupFeature to fail. Not sure what the best way to solve this without just adding more arguments

singjc replied:
You're getting this error?

> AttributeError: 'NoneType' object has no attribute 'loadTransitionGroupFeature'

jcharkow replied:
yes thats the error. I think it is because there is no XIC data/loader object passed

singjc replied:
I passed the mzml loader object now instead to get the feature data from the results file. I think I initially used the sqMass workflow that's why it failed.

jcharkow reviewed (changes requested):
Changes are good. However currently there is an issue in the raw peak GUI display failing to find features. Should try to address this in this PR as well because it concerns these. Unsure if we should use the `plotChromatogram` method I added to the loaders or decouple this feature reading from the peak picking server a bit because it is not doing peak picking just reading results and thus needs different arguments from the other peak pickers (e.g. transitionGroupFeatures).

singjc committed:
[RENAME] feature file peak pickign method

singjc committed:
FIX![GUI]: Feature File Peak Boundaries for raw GUI workflow

Feature file peak boundary rendering doesn't work for the raw GUI
workflow. Added the mzml loader object to 1Dplotting to get feature file
boundaries.

jcharkow committed:
Merge branch 'dev' of github.com:Roestlab/massseer into patch/ui_rename_results_peak_boundaries

jcharkow reviewed (approved):
Good for now however I do think this needs refactoring. I am thinking of separating out FeatureFile from the rest of peak picking because it is inherently different. Will create a separate branch for that though
